AmazonChimeSDKDemo/Podfile

```diff
@@ -0,0 +1,14 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '10.0'
+
+target 'AmazonChimeSDKDemoPods' do
+  use_frameworks!
+  # Pods for AmazonChimeSDKDemoPods
+  pod 'AmazonChimeSDK-Bitcode'
```

Comment:
Should we set the version for ```AmazonChimeSDK-Bitcode```?

Reply:
It will use the latest version, which should be good.

Reply:
Synced up offline, we may encounter building issue when build a previous commit since the Poofile will always pickup the latest SDK version, Ziyi will look into set a fixed SDK version.